docs/UserGuide.md

```diff
@@ -66,80 +79,124 @@ Example:
-To clear the recipe book without the confirmation, users simply have to include the prefix `-f` after the `clear` 
-command <br>
 
-####Clear:
-Usage: `clear` <br>
+Usage: `clear`
 
 The program would then prompt the user to confirm. Type in `yes` to clear and `no` to cancel the clear request.
 
 Example: 
 - clear
     - This would generate a clear request that requires a confirmation. To confirm to clear the recipe book, type in 'yes',
     - else, type in 'no'.
-  
-####Forced Clear:
-Usage: `clear <-f: Prefix for forced Clear>` <br>
+
+<div markdown="block" class="alert alert-info">
+
+️:heavy_exclamation_mark: :computer: To clear the recipe book without the confirmation, users simply have to include the prefix `-f` after the `clear` command. McKitchen is only able to restore the default sample recipes by running `reset`. Currently, McKitchen isn't able to undo actions in the application so use this feature with caution! 
+
+</div>
+
+#### Forced Clear:
+Usage: `clear -f` <br>
 Example:
 - clear -f
     - This would clear the recipe book without having the need for any confirmation.
-    
-Usage: `view -x <recipe index>` <br>
-Example:
-- view -x 1
-    - This would display the full contents of the first recipe in the list which includes its ingredients and steps to prepare the dish.
 
-###### [return to table of content](#table-of-content)
+[<sub>return to table of content<sub>](#table-of-content)
 
-### Store recipe
-Store the recipe in a human-readable text file on the user's computer and is modifiable through a text editor if the user is familiar with the format (JSON). <br>
+### Save recipes
+Save the recipes in a human-readable text file on the your computer and is modifiable through a text editor if the user is familiar with the format (JSON). <br>
 Usage: (Automatically updates text file upon each modifying (writing) operation).
 
-###### [return to table of content](#table-of-content)
+<div markdown="block" class="alert alert-info">
+
+️:information_source: Recipes would be saved in a folder called "data" from where McKitchen is located. <br>
+Example: If McKitchen is saved in the "Downloads" folder, recipes would be saved in a folder called "data" in "Downloads" as "recipebook.json".
+
+</div>
+
+[<sub>return to table of content<sub>](#table-of-content)
 
 ### Load recipe
 Load existing recipes from the text file on the user's computer. Modifications to this file would be displayed accordingly as long as the format is followed. <br>
 Usage: (Automatically loads the recipes upon launching the application).
 
+<div markdown="block" class="alert alert-info">
+
+️:information_source: Recipes would be loaded from a file called "recipebook.json" in a folder called "data" from where McKitchen is located. <br>
+Example: If McKitchen is stored in the "Downloads" folder, recipes would be loaded in a folder called "data" which should "recipebook.json". <br><br>
```

Comment:
perhaps add the file paths to make it easier for dev readers to understand? e.g. /downloads, /download/data/recipebook.json

Reply:
For this part I just added the :computer: which I think is a good comprpomise